catalog/ui/search-ui/search-endpoint/pom.xml

```diff
@@ -388,9 +388,29 @@
                         </Private-Package>
                         <Export-Package/>
                         <Import-Package>
+                            ddf.action,
+                            ddf.catalog,
```

Comment:
@jckilmer is correct. Is there a reason why we can't just do:
```
org.joda.convert;resolution:=optional,
org.eclipse.jetty.*; version="[7.6.0,9.0.0)",
*
```
Assuming the dependencies are listed correctly, this should work. I'd like to get away from listing out all of the imports manually.